src/DurableTask.SqlServer/Scripts/logic.sql

```diff
@@ -65,8 +65,8 @@ BEGIN
     DECLARE @taskHub varchar(50) = __SchemaNamePlaceholder__.CurrentTaskHub()
     DECLARE @now datetime2 = SYSUTCDATETIME()
 
-    DECLARE @liveInstances int = 0
-    DECLARE @liveTasks int = 0
+    DECLARE @liveInstances bigint = 0
+    DECLARE @liveTasks bigint = 0
 
     SELECT
         @liveInstances = COUNT(DISTINCT E.[InstanceID]),
```

Comment:
Now that you're making these local variables into `bigint`, do you need to also change `COUNT` to [`COUNT_BIG`](https://learn.microsoft.com/en-us/sql/t-sql/functions/count-big-transact-sql?view=sql-server-ver16)?